dev/FARFF/test_FARFF_AuxMaps.py

```diff
@@ -10,7 +10,6 @@
 #from ppafm import fieldOCL     as FFcl
 #from ppafm import RelaxOpenCL  as oclr
 #from ppafm import common       as PPU
-#from ppafm import basUtils
 #from ppafm.AFMulatorOCL_Simple  import AFMulator
 #from ppafm.GeneratorOCL_Simple2 import InverseAFMtrainer
 #from ppafm.AuxMap import AuxMaps
```

Comment:
Please remove the unnecessary comments and `sys.path.append`. Same for all the occurrences below.